### ISSUE TITLE
[Suggestion] Allow to be able to drop pkhex files onto PKMN-NTR and it'll automatically inject the pokemon into the game

### ISSUE THREAD

**Issue by Naturally-Delicious** [drgoku282/PKMN-NTR#122]
I remember in an early release (I know the release-description said that "it'll be the last stable release for a while") we were able to drop 1+ pkhex files in the "Write File" tab and the program would automatically inject the Pokemon depending on which box we wanted to start on and we were able to "increment", meaning the ox number would go up as we injected Pokemon. In later releases, you are  still able to drop a pkhex file, but only one at a time, and you have to click "write" which can be time-consuming if you want to inject a lot of Pokemon. One reason that I thought that might be the reason as to why the old inject feature was removed is bc it was too dangerous? Like people might've accidentally erased Pokemon when dropping in files? IDK, but if that was the reason, maybe make a shortcut that would enter the program into "unsafe" mode, or something like that and unlock potentially dangerous things? I've had this peeve for a while now, but I didn't wanna say anything cus I could just stay on an older version os pkmn-ntr, but with USUM coming out, I practically HAVE to move up to the later releases. The old inject method is the only reason I use PKMN-NTR

tl;dr Allow us to be able to drop multiple PkHEX files into PKMN-NTR and be able to inject them all at once into the game. (used to be an old feature but it's been removed.) 

TY for reading this mess thank u bye.

EDIT: The release I'm talking about is 2.4 - Legendary RNG Seed dumping.

**Comment by drgoku282** [comment]
The feature was removed because it was insecure, first because the overwritting of other pokémon, and the  second and main reason is because the program might write data in dangerous areas. This program wasn't designed to work with a lot of data, so it's safer to use PKHeX to do heavy data manipulation.
